x/liquidity/spec/02_state.md

```diff
@@ -1,105 +1,117 @@
+    Msg        MsgDepositWithinBatch
 }
 ```
 
-`WithdrawMsgState` defines the state of withdraw message that contains state information as it is processed in the next batch(s)
+## WithdrawMsgState for Batch Messages
+
+The `WithdrawMsgState` state defines the state of the withdraw message that contains state information as it is processed in the next batch or batches.
 
 ```go
 type WithdrawMsgState struct {
-	MsgHeight  uint64 // height where this message is appended to the batch
-	MsgIndex   uint64 // index of this withdraw message in this liquidity pool
-	Executed   bool   // true if executed on this batch, false if not executed yet
-	Succeeded  bool   // true if executed successfully on this batch, false if failed
-	ToBeDelete bool   // true if ready to be deleted on kvstore, false if not ready to be deleted
-	Msg        MsgWithdrawWithinBatch
+    MsgHeight  uint64 // block height where this message is appended to the batch
+    MsgIndex   uint64 // index of this withdraw message in this liquidity pool
+    Executed   bool   // true if executed on this batch, false if not executed
+    Succeeded  bool   // true if executed successfully on this batch, false if failed
+    ToBeDelete bool   // true if ready to be deleted on kvstore, false if not ready to be deleted
+    Msg        MsgWithdrawWithinBatch
 }
 ```
 
-`SwapMsgState` defines the state of swap message that contains state information as it is processed in the next batch(s)
+## SwapMsgState for Batch Messages
+
+`SwapMsgState` defines the state of swap message that contains state information as it is processed in the next batch or batches.
 
 ```go
 type SwapMsgState struct {
-	MsgHeight          uint64 // height where this message is appended to the batch
-	MsgIndex           uint64 // index of this swap message in this liquidity pool
-	Executed           bool   // true if executed on this batch, false if not executed yet
-	Succeeded          bool   // true if executed successfully on this batch, false if failed
-	ToBeDelete         bool   // true if ready to be deleted on kvstore, false if not ready to be deleted
-	OrderExpiryHeight  int64  // swap orders are cancelled when current height is equal or higher than ExpiryHeight
-	ExchangedOfferCoin sdk.Coin // offer coin exchanged until now
-	RemainingOfferCoin sdk.Coin // offer coin currently remaining to be exchanged
-	Msg                MsgSwapWithinBatch
+    MsgHeight          uint64 // block height where this message is appended to the batch
+    MsgIndex           uint64 // index of this swap message in this liquidity pool
+    Executed           bool   // true if executed on this batch, false if not executed
+    Succeeded          bool   // true if executed successfully on this batch, false if failed
+    ToBeDelete         bool   // true if ready to be deleted on kvstore, false if not ready to be deleted
+    OrderExpiryHeight  int64  // swap orders are cancelled when current height is equal or greater than ExpiryHeight
```

Comment:
correct me if wrong, but even for height references I think the correct word here is "greater" when we use equal to 
(is there a blockchain use of "higher than" that I need to learn about?)

Reply:
You are right. It should be `greater`.